Godeps/_workspace/src/github.com/contiv/contivmodel/network.json

```diff
@@ -17,6 +17,13 @@
 					"title": "Tenant Name",
 					"length": 64
 				},
+				"nwType": {
+					"type": "string",
+					"format": "^(infra|data)$",
+					"default": "\"data\"",
```

Comment:
Yes, need the \". Just "default": "data" results in nwType = data - i.e without quotes as if data is a number.